subprojects/testing-base/src/integTest/groovy/org/gradle/testing/SecurityManagerIntegrationTest.groovy

```diff
@@ -21,11 +21,14 @@ import org.gradle.integtests.fixtures.AbstractIntegrationSpec
 import org.gradle.integtests.fixtures.JUnitXmlTestExecutionResult
 import org.gradle.integtests.fixtures.timeout.IntegrationTestTimeout
 import org.gradle.util.GradleVersion
+import org.gradle.util.Requires
+import org.gradle.util.TestPrecondition
 import spock.lang.IgnoreIf
 
 import static org.gradle.util.Matchers.containsText
 import static org.gradle.util.Matchers.matchesRegexp
 
+@Requires(TestPrecondition.JDK16_OR_EARLIER) // SecurityManager is deprecated in Java 17: https://openjdk.java.net/jeps/411
```

Comment:
can we make this so the test process uses <JDK17? I think we don't want to lose coverage for this when we move to JDK17 in our pipeline